### PR TITLE
Added go2

### PR DESCRIPTION
**Pull request by moorepants**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by moorepants** [comment]
@conda-forge/python I need some help on this recipe. This package uses the `data_files` keyword argument in the `setup.py` to place files in `/usr/lib', `/usr/bin/`, and '/usr/share`. I patched the `setup.py` file such that these are placed into `$PREFIX/lib`, `$PREFIX/bin`, and `$PREFIX/share` but they don't seem to be installed there in the test environment. Is there something I need to do so that conda is aware of these files and that they ultimately get installed in the proper location?

**Comment by moorepants** [comment]
Just realized this depends on an import of `gi.repository`.

**Comment by moorepants** [comment]
Maybe `gi.repository` is from the pygobject package.

**Comment by moorepants** [comment]
Thanks @isuruf!

**Comment by moorepants** [comment]
Ok @conda-forge/python I think this is ready to merge.

**Comment by moorepants** [comment]
I noticed that this requires the unix cmd tool "tree" in the source code. But there is no tree feedstock. How should I handle that?

**Comment by isuruf** [comment]
You should add a recipe for that.

**Comment by moorepants** [comment]
Waiting on #11495.

**Comment by moorepants** [comment]
@conda-forge-admin please rerender

**Comment by moorepants** [comment]
@conda-forge/help-python This is ready to be merged now. I'll work with the package author to try to get a more portable version for future builds.

File: recipes/go2/build.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+$PYTHON -m pip install . -vv
+# /usr/lib/ is hardcoded in a number of files, so replace with the build
+# prefix, which will then be replaced on install with the install prefix.
+sed -i "s|/usr/lib/go2|$PREFIX/lib/go2|g" $PREFIX/bin/go2
+sed -i "s|/usr/lib/go2|$PREFIX/lib/go2|g" $PREFIX/lib/go2/go2.sh
+sed -i "s|/usr/lib/go2|$PREFIX/lib/go2|g" $PREFIX/lib/go2/go2.py
+sed -i "s|/usr/lib/go2|$PREFIX/lib/go2|g" $PREFIX/share/man/man1/go2.1
+sed -i "s|/usr/bin/python3|$PYTHON|g" $PREFIX/lib/go2/go2.py
```
[the remark this answers is not in the document]
Comment (moorepants) [reply recipes/go2/build.sh:8]: No, I guess it is working correctly.

**Comment by moorepants** [comment]
@conda-forge/help-python 

Checking in again, I think this is ready to be merged. I can iterate more once it is a feedstock and we get some changes merged upstream.

File: recipes/go2/COPYING [hunk 2]
```diff
@@ -0,0 +1,340 @@
+		    GNU GENERAL PUBLIC LICENSE
```
Comment (isuruf) [inline recipes/go2/COPYING:1]: Is this file not in the source archive?

Comment (moorepants) [reply recipes/go2/COPYING:1]: No, see this issue for all changes I'm requesting for the upstream package: https://bitbucket.org/DavidVilla/go2/issues/3/suggestions-to-improve-downstream

**Comment by moorepants** [comment]
Thanks for the review and help @isuruf!

**Pull request merged**